dbt-cta/action_builder/models/sources.yml

```diff
@@ -1,72 +1,13 @@
 version: 2
 sources:
-- name: cta
+- name: tmc_action_builder
   database: "{{ env_var('CTA_PROJECT_ID') }}"
   schema: "{{ env_var('CTA_DATASET_ID') }}"
+  quoting:
+    database: true
+    schema: false
+    identifier: false
   tables:
-  - name: _airbyte_raw_action_assignment_group_users
```

Comment:
The `_airbyte_raw` tables should stay in the sources

Reply:
Gotcha, thanks!